src/domain/event.ts

```diff
@@ -0,0 +1,119 @@
+import { Address, BigInt, ByteArray, Bytes, crypto } from '@graphprotocol/graph-ts';
+import { Event, Proposal, ReputationHolder } from '../types/schema';
+import { concat } from '../utils';
+
+export function addNewDAOEvent(avatar: Address, daoName: string, timestamp: BigInt): void {
```

Comment:
how about add a function addEvent(address, name,id...) .
- so no need specific function and duplicate code for each event

Reply:
done